tests/distributions/test_continuous.py

```diff
@@ -549,6 +549,20 @@ def test_studentt_logp(self):
             lambda value, nu, mu, sigma: st.t.logpdf(value, nu, mu, sigma),
         )
 
+    def test_skewstudentt_logp(self):
+        check_logp(
+            pm.SkewStudentT,
+            R,
+            {"a": Rplus, "b": Rplus, "mu": R, "lam": Rplus},
+            lambda value, a, b, mu, lam: st.t.logpdf(value, a, b, mu, lam**-0.5),
+        )
+        check_logp(
```

Comment:
No need to test logp/logcdf twice just for the sigma/tau parametrization. These are the most expensive tests 

The helper to test RVs directly (which is missing) is better for just making sure tau gets converted into the expected sigma

Also missing the support_point test